packages/seacas/libraries/aprepro_lib/apr_builtin.cc

```diff
@@ -179,7 +179,7 @@ namespace SEAMS {
     return (temp);
   }
 
-  // DO_HYPOT: calcluate sqrt(p^2 + q^2)
+  // DO_HYPOT: calculate sqrt(p^2 + q^2)
```

Comment:
C++ has `std::hypot` now.

Reply:
That's true... I will look into what can be replaced with C++11 math/numeric functions.